packages/terra-base/tests/jest/Base.test.jsx

```diff
@@ -8,6 +8,7 @@ it('should support rendering a string without translation', () => {
   expect(base).toMatchSnapshot();
 });
 
+
```

Comment:
nit:  added an extra line here not needed

Reply:
Removed in this commit: https://github.com/cerner/terra-core/pull/1012/commits/de92c42b10d391c997e2d2edd3ee4b606e679213